src/renderer/components/AccountListSidebar/AccountHoverInfo.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { Component, useEffect, useState } from 'react'
+import { C, T } from '@deltachat/jsonrpc-client'
+import { filesize } from 'filesize'
+import debounce from 'debounce'
+
+import { BackendRemote, onDCEvent } from '../../backend-com'
+import { getLogger } from '../../../shared/logger'
+import { runtime } from '../../runtime'
+
+import styles from './styles.module.scss'
+
+const log = getLogger('AccountListSidebar/AccountHoverInfo')
+
+export default function AccountHoverInfo({
+  account,
+  isSelected,
+}: {
+  account: T.Account
+  isSelected: boolean
+}) {
+  const tx = window.static_translate
+  const [bgSyncDisabled, setBgSyncDisabled] = useState<boolean>(false)
+  useEffect(() => {
+    runtime.getDesktopSettings().then(({ syncAllAccounts }) => {
+      setBgSyncDisabled(!syncAllAccounts)
+    })
+  }, [account.id])
+
+  const showConnection = isSelected || !bgSyncDisabled
+
+  let content: JSX.Element
+
+  if (account.kind === 'Unconfigured') {
+    content = (
+      <>
+        <b>{tx('unconfigured_account')}</b>
+        <div className={styles.hoverInfoProperty}>
+          {tx('unconfigured_account_hint')}
+        </div>
+      </>
+    )
+  } else {
+    content = (
+      <>
+        <b>{account.displayName ? account.displayName : account.addr}</b>
+        {showConnection && (
+          <div className={styles.hoverInfoProperty}>
+            <Connectivity accountId={account.id} />
+          </div>
+        )}
+        {bgSyncDisabled && (
+          <div className={styles.hoverInfoProperty}>
+            ⏻ {tx('background_sync_disabled_explaination')}
+          </div>
+        )}
+      </>
+    )
+  }
+
+  return (
+    <div className={styles.accountHoverInfo} role='tooltip'>
+      {content}
+      <div className={styles.hoverInfoFooter}>
+        <AccountSize accountId={account.id} /> - {account.id}
```

Comment:
this "number" looks like a bug :) we should not show debug info that are not meaningful at all for the user in primary UI.

```suggestion
        <AccountSize accountId={account.id} />
```

i am also not sure about showing "size" that prominently, but i'd give that a try. it is useful in general and also somehow clear in meaning

Reply:
> i am also not sure about showing "size" that prominently, but i'd give that a try

I want to eventually have a storage size section in the settings where you can manage storage usage kinda like telegram has.

Maybe it would make sense to move the account id to the about dialog, since we also have the dc version there.
It can be useful for understanding logs that people send us and to find the account folder on disk.

Reply:
The account size is also interesting to differentiate accounts when you have the same one multiple times, that was one of the main reasons why I added it as a tooltip in the old account switcher.

Reply:
made a pr that updates the about dialog to show the selected account id and to show more versions (core, electron, node - wanted to do that for a longer time already anyway)